tests/src/test/scala/org/scalanative/bindgen/BindgenSpec.scala

```diff
@@ -18,15 +18,21 @@ class BindgenSpec extends FunSpec {
     }
 
     def bindgen(inputFile: File, name: String, outputFile: File): Unit = {
-      Bindgen()
+      val result = Bindgen()
         .bindgenExecutable(new File(bindgenPath))
         .header(inputFile)
         .name(name)
         .link("bindgentests")
         .packageName("org.scalanative.bindgen.samples")
         .excludePrefix("__")
         .generate()
-        .writeToFile(outputFile)
+
+      result match {
+        case Right(binding) =>
+          binding.writeToFile(outputFile)
```

Comment:
My suggestion is to eventually remove error emitting code from these tests so we can add `assert(binding.errors.isEmpty)` here.

Reply:
But we still want to check what bindings are generated if a header file cannot be perfectly converted to Scala code.
For example we check that literal define is skipped if its value does not fit into Scala type.
Even if we check in `BindgenReportingSpec` that warning is printed for this case it does not guarantee that the code is not outputted.

Reply:
Assuming these are edge cases it could still make sense to have them in the reporting spec.